### PR TITLE
es-ES: Added STR 6268 && 6269, applies to #1596

### PR DESCRIPTION
**Pull request by frenchiveruti**
If you are applying translations for an issue (or multiple), please list them below (if not, delete this text).

Applying for issue(s):
- #1596

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,5 +1,6 @@
 # en-GB is maintained in main repository
 data/language/en-GB.txt
+gitcode.txt
```
Comment (Gymnasiast) [inline .gitignore]: This doesn't belong in this PR.

**Comment by frenchiveruti** [comment]
All clear for merge.

**Comment by Gymnasiast** [comment]
I meant that you should have reverted the changes to the file, not deleting it completely. Could you please restore it, so that "Files changed" only shows the changed line in es-ES.txt?

**Comment by frenchiveruti** [comment]
Sorry git is not my strong you know that haha.
